tests/test_build.py

```diff
@@ -32,21 +32,35 @@ def test_build_e2e(tmp_path: Path):
             "--truncation",
         ],
         cwd=tmp_path,
+        capture_output=True,  # Add this
+        text=True,  # Add this to get strings instead of bytes
     )
 
-    assert result.returncode == 0
+    # Print the output to see what's failing
+    if result.returncode != 0:
```

Comment:
I fixed this in the other query PR, I think it's a bit different than this fix. Basically returncode is 0 as long as the subprocess itself is happy so you parse the output text unconditionally